aptiblejs/src/analytics.js

```diff
@@ -76,16 +76,20 @@ export function fireAllPixels() {
 }
 
 export function firePixel(service) {
-  switch (service) {
-    case 'twitter':
-      if (window.twttr) {
-        twttr.conversion.trackPid('nz8pr', { tw_sale_amount: 0, tw_order_quantity: 0 });
-      }
-      break;
-    case 'quora':
-      if (window.qp) {
-        window.qp('track', 'Generic');
-      }
-      break;
+  try {
+    switch (service) {
+      case 'twitter':
+        if (window.twttr) {
+          twttr.conversion.trackPid('nz8pr', { tw_sale_amount: 0, tw_order_quantity: 0 });
+        }
+        break;
+      case 'quora':
+        if (window.qp) {
+          window.qp('track', 'Generic');
+        }
+        break;
+    }
+  } catch(e) {
+    // Prevent analytics from throwing exceptions
```

Comment:
What expected error are you silencing here?  It would probably be a good idea to only silence the error you expect, and re-throw any exceptions that aren't expected.  That could save you some headache in the future.

Reply:
As discussed on Slack, added in a console.warn for now rather than hiding all exceptions.